### DISCUSSION TITLE
finite rate chemistry for methane

### DISCUSSION THREAD

**Discussion by aditya14172** [firemodels/fds#13359] [General]
facing issue with finite rate chemistry of methane, the increase in temperature is not enough and also ignition is not sustaining

[methane.txt](https://github.com/user-attachments/files/16752830/methane.txt)

**Comment by drjfloyd** [comment]
How much OXYGEN have you defined as being present in your domain?

**Reply by aditya14172** [reply]
21%, same as the amount of oxygen present in atmosphere.

**Reply by drjfloyd** [reply]
Where in your input file did you define that the SPEC of OXYGEN has an initial volume fraction of 21 %?

**Reply by aditya14172** [reply]
Sir by default for air it is defined: &SPEC ID='AIR', BACKGROUND=T,
SPEC_ID(1)='OXYGEN', VOLUME_FRACTION(1)=0.2076,
SPEC_ID(2)='NITROGEN', VOLUME_FRACTION(2)=0.7825,
SPEC_ID(3)='WATER VAPOR', VOLUME_FRACTION(3)=0.0095,
SPEC_ID(4)='CARBON DIOXIDE', VOLUME_FRACTION(4)=0.0004 / like for the above case if i see in slice file for oxygen i can see the variation of oxygen volume fraction
please suggest any other error i might be making

**Reply by drjfloyd** [reply]
"The array SPEC_ID_NU contains the list of tracked species participating in the reaction as a product or
reactant."

Your REAC uses the tracked species OXYGEN. Not the lumped background species AIR.

**Reply by aditya14172** [reply]
Sir, i have tried that case too , even that is not working, So referring the User Guide i have tried this format of defining SPEC_ID_NU

**Comment by mcgratta** [comment]
Take a look at the [Smyth Slot Burner](https://github.com/firemodels/fds/tree/master/Validation/Smyth_Slot_Burner/FDS_Input_Files) validation cases. Run them, and then use this input file for your own case.

**Reply by aditya14172** [reply]
Sir, Smyth Slot Burner has infinite rate chemistry for R1. I need to learn what should I do for single step finite rate chemistry.

**Comment by mcgratta** [comment]
What case have you tried that you say is not working? What were the results of your effort?

**Reply by aditya14172** [reply]
Sir,here is the case, getting the similar results: 
[methane.txt](https://github.com/user-attachments/files/16763187/methane.txt)

**Comment by mcgratta** [comment]
The case `Smyth_AM_fine.fds` has a three step reaction, one fast, two finite-rate.

**Reply by aditya14172** [reply]
Yes sir, but here i am trying for a single step finite rate in case of methane.

**Reply by mcgratta** [reply]
Well, I've never tried to run a single step finite rate simulation of methane combustion. I cannot say whether your kinetic parameters are appropriate or not. I cannot say whether your grid is resolved enough.

**Comment by drjfloyd** [comment]
Please carefully read the User's Guide and look at the examples that use Arrhenius chemsitry and also review how species are handeled in FDS. 

As stated in the User's Guide and as I posted here earlier: "The array SPEC_ID_NU contains the list of tracked species participating in the reaction as a product or reactant."

&REAC ID = 'R1'
FUEL = 'METHANE'
A = 5.012e16
E = 151080
SPEC_ID_NU = 'METHANE','OXYGEN','CARBON DIOXIDE','WATER VAPOR'
NU = -1,-2,1,2
SPEC_ID_N_S = 'METHANE','OXYGEN'
N_S = 0.1,1.65 /

R1 is not using AIR. R1 is using OXYGEN.

Here is your input for the tracked species of oxygen:
&SPEC ID='OXYGEN' /

You have not given an initial concentration.
